cookbook/core/type_system/structured_dataset.py

```diff
@@ -151,19 +152,19 @@ def bq_to_pandas(sd: StructuredDataset) -> pd.DataFrame:
 
 
 # %%
-# Numpy Encoder and Decoder
+# NumPy Encoder and Decoder
 # ==========================
 #
-# ``StructuredDataset`` ships with an encoder and a decoder that handles conversion of a Python value to Flyte literal and vice-versa, respectively.
-# Let's understand how they need to be written by defining a Numpy encoder and decoder, which in turn helps use Numpy array as a valid type within structured datasets.
+# ``StructuredDataset`` ships with an encoder and a decoder that handles conversion of a Python value to a Flyte literal and vice-versa, respectively.
+# Let's understand how to write to write them by defining a NumPy encoder and decoder, which helps use NumPy array as a valid type within structured datasets.
 
 
 # %%
-# Numpy Encoder
+# NumPy Encoder
 # ^^^^^^^^^^^^^
 #
 # We extend ``StructuredDatasetEncoder`` and implement the ``encode`` function.
-# The ``encode`` function converts Numpy array to an intermediate format like parquet.
+# The ``encode`` function converts NumPy array to an intermediate format like parquet.
```

Comment:
Typo. "The ``encode`` function converts a NumPy..."

Ambiguous, "like parquet". Is parquet an example in this case or is this what your sample code actually chooses to do?

Reply:
If I understand this right, we can specify the intermediate format here, and we have specified that as 'parquet'.

Reply:
Does this look good? "The ``encode`` function converts NumPy array to an intermediate format (parquet file format in this case)."